test/integ/pandas/test_pandas_tools.py

```diff
@@ -142,6 +142,7 @@ def test_write_pandas_with_overwrite(
 @pytest.mark.parametrize("quote_identifiers", [True, False])
 @pytest.mark.parametrize("auto_create_table", [True, False])
 @pytest.mark.parametrize("create_temp_table", [True, False])
+@pytest.mark.parametrize("table_type", ["", "temp", "temporary", "transient"])
```

Comment:
Please move this test into it's own test. This test is very thorough and it's one of the slowest one. Let's try to not add more checks to it.
On the bright side I think for your test you can skip a bunch of checks that this one does.